packages/devtools_app/lib/src/screens/inspector_v2/widget_properties/properties_view.dart

```diff
@@ -0,0 +1,166 @@
+// Copyright 2024 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:devtools_app_shared/ui.dart';
+import 'package:flutter/material.dart';
+
+import '../../../shared/diagnostics/diagnostics_node.dart';
+import '../../../shared/diagnostics_text_styles.dart';
+import '../inspector_controller.dart';
+
+class PropertiesView extends StatelessWidget {
+  const PropertiesView({
+    super.key,
+    required this.controller,
+    required this.node,
+  });
+
+  final InspectorController controller;
+  final RemoteDiagnosticsNode node;
+
+  @override
+  Widget build(BuildContext context) {
+    return ValueListenableBuilder<List<RemoteDiagnosticsNode>>(
+      valueListenable: controller.selectedNodeProperties,
+      builder: (context, properties, _) {
+        return Container(
+          margin: const EdgeInsets.all(denseSpacing),
+          child: ListView.builder(
+            itemCount: properties.length,
+            itemBuilder: (context, index) {
+              return PropertyItem(
+                index: index,
+                properties: properties,
+              );
+            },
+          ),
+        );
+      },
+    );
+  }
+}
+
+class PropertyItem extends StatelessWidget {
+  const PropertyItem({
+    super.key,
+    required this.properties,
+    required this.index,
+  });
+
+  final List<RemoteDiagnosticsNode> properties;
+  final int index;
+
+  @override
+  Widget build(BuildContext context) {
+    final property = properties[index];
+    final theme = Theme.of(context);
+
+    return DecoratedBox(
+      decoration: BoxDecoration(
```

Comment:
wdyt about removing this radius and just letting the property editor fill the container of the splitter's child? I worry in embedded mode we will have too many borders & too much padding that take up space and make the interface busy. If this list just runs up to the edge of the main rounded border that the entire details view sits in, that may be a better look for small windows

Reply:
Makes sense! The list doesn't quite run up the edge of the main rounded border, but I like that it now matches the layout explorer which is also square. Let me know what you think:

<img width="1266" alt="Screenshot 2024-07-23 at 4 10 08 PM" src="https://github.com/user-attachments/assets/1a122431-a438-4d38-9d1a-ac5070c9121f">

Reply:
it looks like all the shaded rows have a double border. Can we instead just put a border between list items and then have the whole list have a single rectangular border?

Reply:
Made a few more changes here based on our conversation yesterday to separate the RenderObject properties from the Widget properties. I first tried a split view like we use in the debugger panel for the variables, callstack, etc. but that ended up looking very strange:

![flex_table](https://github.com/user-attachments/assets/21709757-e4d0-4e6d-b35d-ea62fc4bab38)

So I switched to a tab view. Let me know what you think! The only issue is that when the last row is at the bottom of the table, it has a double border. I'm not sure how to avoid that, besides getting rid of the last border on that row, but that looks bad if the table doesn't take up all the space:

<img width="425" alt="Screenshot 2024-07-25 at 9 40 56 AM" src="https://github.com/user-attachments/assets/f3cf0b5c-939f-4978-aa12-d999f843e6bf">